src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/items/LumberAxe.java

```diff
@@ -19,6 +19,11 @@
 import me.mrCookieSlime.Slimefun.Setup.SlimefunManager;
 import me.mrCookieSlime.Slimefun.api.SlimefunItemStack;
 
+@Override
+protected boolean areItemHandlersPrivate() {
+	return false;
```

Comment:
Please update the formatting on this and move the method further down, it shouldn't be at the top.

Reply:
changed it